lib/solidus_feeds/generators/google_merchant.rb

```diff
@@ -0,0 +1,60 @@
+# lib/google_merchant.rb
+
+require 'builder'
+
+module SolidusFeeds
+  module Generators
+    class GoogleMerchant
+      attr_accessor :products
+
+      def initialize(products)
+        self.products = products
+      end
+
+      def call
```

Comment:
Not sure if this is this is done intentionally this way for the sake of development. Just want to remind that this is supposed to accept an external IO object and be like `def call(io)` or `def call(stream)` in the final version.